docs/reference/data-streams/set-up-a-data-stream.asciidoc

```diff
@@ -20,11 +20,11 @@ and its backing indices.
 === Prerequisites
 
 * {es} data streams are intended for time-series data only. Each document
-indexed to a data stream must contain a shared timestamp field.
+indexed to a data stream must contain the `@timestamp` field. This field must be
+mapped as a <<date,`date`>> or <<date_nanos,`date_nanos`>> field data type.
 +
-TIP: Data streams work well with most common log formats. While no schema is
-required to use data streams, we recommend the {ecs-ref}[Elastic Common Schema
-(ECS)].
+TIP: We recommend using the {ecs-ref}[Elastic Common Schema
+(ECS)], which includes the `@timestamp` field.
```

Comment:
This seems strange to say that we recommend `@timestamp`, since we require it to be `@timestamp` currently